app/views/performance/index.html.haml

```diff
@@ -0,0 +1,20 @@
+- content_for :page_title, "Performance"
+
+%h1= yield :page_title
+
+%h2 Comments on planning applications
+
+%table
+  %caption
+    New and returning people commenting on applications over time
+  %thead
+    %tr
+      %th Date
+      %th First time commenters
+      %th Returning commenters
+  %tbody
+    - @daily_comment_metrics.each do |day|
```

Comment:
While this collection could be extracted into a partial, I think it's actually more helpful to have all the table markup inline, since it's such a simple table.